manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: LingitAS.Lingdys4
+PackageVersion: 4.5.111.0
+PackageLocale: nb-NO
+Publisher: Lingit AS
+PublisherUrl: https://lingit.no/
+PublisherSupportUrl: https://lingit.no/
+PrivacyUrl: https://lingit.no/

```

Comment:
Please ensure this is a direct link to the privacy policy, not just to the package homepage